libnethack/src/sounds.c

```diff
@@ -810,6 +810,26 @@ domonnoise(struct monst *mtmp)
         else
             verbl_msg = "Who do you think you are, War?";
         break;
+    case MS_RTP:
+        {
+            static const char *const rtp_foe_msg[3] = {
```

Comment:
Don't specify array size when initializing with `{}`. It is inferred automatically.